assets/js/wpr-cpcss.js

```diff
@@ -0,0 +1,104 @@
+var $ = jQuery;
```

Comment:
What's our take on using jQuery or not? Should we avoid it since it's an "of-the-past" framework? What happens if someone decides to unload it from the admin?

@hellofromtonya @crystinutzaa

Reply:
We should eliminate the use of jQuery and standardize on vanilla JS or React.

Reply:
Ok. So vanilla JS sounds good at this point?
I will convert the code today and get rid of all jQuery.

Reply:
This is done. No more jQuery only Vanilla JS.